### ISSUE TITLE
Save and display Issue Boards (Kanban, Projects) activities on the "activity" page

### ISSUE THREAD

**Issue by MrCatEsq** [go-gitea/gitea#19178]
### Feature Description

What do you think about to add Projects (Issue Boards, Kanban boards) activities to the history of the repo (in "activity" page or somewhere else)? It will be very useful to project managers, team lead developers, product owners, etc. Maybe Lanre @adelowo (who [made](https://github.com/go-gitea/gitea/pull/8346) possible project management in Gitea 1.13) or someone else will find it interesting to implement this feature? 

For example: 
23:30 user Anna added the card "bug #2231" to the Don board of the "MyProject" project; 
23:29 user Bob moved the card "bug #2231" to the board Testing the project "MyProject"; 
23:25 User Bob changed the description in the "bug #2231" card to "something done"
... 
In short, the same thing that we now see in the Activity section for commits, branches and issues, but for projects, milestones, boards, task creations, attached tags (topics), labels, assignees, due dates, comments, cards.

This is already implemented in specialized task managers such as Trello, Asana and others. I understand that project management isn't the main goal of Gitea, but for many users this feature will make possible to use one tool instead of two. It would be very convenient: to use one product for version control, project management, documentation (wiki), deployment. Like "simple and lightweight Gitlab". This feature is also realized on Github as Projects, I attached an example.

Some time ago I asked this feature in discord server and as I understood, now there is no such functionality built-in and impossible to see these activities via UI or API.

### Screenshots

**Github example:**
![Github implementation example](https://user-images.githubusercontent.com/95284657/159566655-2e41f497-685e-4db2-bb0f-39c983707658.png)

**Trello example:**
![Trello implementation example](https://www.flowji.com/wp-content/uploads/trello-website-project-02.jpg)

**Asana example**
![Asana implementation example](https://luna1.co/f66a5f.png)

**Gitlab example**
![Gitlab implementation example](https://about.gitlab.com/images/12_10/sort-issue-activity-by-newest-first.png)

**Comment by adelowo** [comment]
This is very much possible. I might be able to take a stab at it sometime at the end of the month I think

**Comment by MrCatEsq** [comment]
@adelowo is there any news about this issue?

**Comment by yakov-eremin** [comment]
Yeah, it will be pretty convenient to see such info. Even history in card card itself (not on "activity" page), like it made in Trello, could be useful.
